CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.7.1] - 2021-08-20
```

Comment:
Setting the version and date happens automatically as part of the release automation, so this line needs to be removed again sorry. A bit too ambitious :)


```suggestion

```